adb_graphics/default_specs.yml

```diff
@@ -922,7 +922,7 @@ emissions: # Smoke emissions
     ticks: 0
     title: Smoke Emissions
     transform: conversions.to_micrograms_per_m3
-    unit: $\mu g/m^3$
```

Comment:
Assuming we don't need to change the conversion calculation, that this is just the preferred convention for the units?  Do we need to change the name of the transform?

Reply:
I'm pretty sure we don't need to change the conversion.  The original units for this field are kg m-2 s-1, so we are really only converting the kg to micrograms.  I don't think it's necessary to change the name of the transform, even though it's a bit of a misnomer, in this case.